README.md

```diff
@@ -14,12 +14,13 @@ We call annotated Go programs Gobra programs and use the file extension `.gobra`
 - Git
 
 ### Installation
+TODO: adapt instructions
 1. Create a folder for your Gobra development. We will refer to this folder as `gobraHome`.
 2. Clone Gobra and Viper dependencies
     - Change directory to `gobraHome`
-    - [silicon](https://github.com/viperproject/silicon) (commit `47e74c64b75f1c186a1307c07f21198e61103334`)
-    - [carbon](https://github.com/viperproject/carbon) (commit `f9c4027cd3bd82f202e7fe78866f8ff794a47232`)
-    - [viperserver](https://github.com/viperproject/viperserver) (commit `4f441dff43bb1ce698fb40fa50fbaf0ed203aec2`)
+    - [silicon](https://github.com/viperproject/silicon) (please check the specific commit to fetch in file `viper-toolchain-versions.sh`)
+    - [carbon](https://github.com/viperproject/carbon) (please check the specific commit to fetch in file `viper-toolchain-versions.sh`)
+    - [viperserver](https://github.com/viperproject/viperserver) (please check the specific commit to fetch in file `viper-toolchain-versions.sh`)
```

Comment:
You should state the commit hash and not delegate the people somewhere else. We want that it is as easy as possible for people to install Gobra.

Reply:
This is a draft PR, it is not meant for review yet!! The readme is marked with a TODO and this will not even be necessary, as we are now using git submodules.